string file_name
float32 pitch       # [m]
-----
string log